src/primitives/arc/mod.rs

```diff
@@ -1,17 +1,14 @@
 //! The arc primitive
 
-mod linear_equation;
-mod plane_sector;
-mod points;
-mod styled;
-
 use crate::{
     geometry::{Angle, Dimensions, Point, Size},
     primitives::{Circle, Primitive, Rectangle},
     transform::Transform,
 };
-pub(in crate::primitives) use linear_equation::LinearEquation;
-pub(in crate::primitives) use plane_sector::{PlaneSector, PlaneSectorIterator};
+
+mod points;
+mod styled;
+
```

Comment:
This should be above all the `use`s for consistency IMO

Reply:
OK, makes sense.